We don't have lua-utf8 in aports yet, so use dromozoa-utf8 instead.

--- a/spec/json_load_spec.lua
+++ b/spec/json_load_spec.lua
@@ -1,5 +1,5 @@
 --luacheck: ignore describe it
-local utf8 = require "lua-utf8" -- luarocks install luautf8
+local utf8 = require "dromozoa.utf8"
 describe('rapidjson.load()', function()
   local rapidjson = require('rapidjson')
   describe('report error', function()
